audio-worklet/design-pattern/awn-shared-buffer/shared-buffer-worker.js

```diff
@@ -132,9 +151,7 @@ function initialize(userConfig) {
 }
 
 onmessage = (eventFromMain) => {
-  switch (eventFromMain.data.message) {
-    case 'INITIALIZE_WORKER':
-      initialize(eventFromMain.data);
-      break;
+  if (eventFromMain.data.message === 'INITIALIZE_WORKER') {
+    initialize(eventFromMain.data.options);
   }
```

Comment:
To be clear, you should handle the case where message isn't 'INITIALIZE_WORKER'.  But as example code, I'm ok if you don't. (But this is going to bite you when this gets updated someday to add more messages for something.)

Reply:
Done.